packages/client/src/v2-events/layouts/sidebar/PerformanceNavigationGroup.tsx

```diff
@@ -47,16 +49,15 @@ export function PerformanceNavigationGroup({
                     id={`navigation_dashboard_${config.id}`}
                     isSelected={currentWorkqueueSlug === 'dashboard'}
                     label={intl.formatMessage(config.title)}
-                    onClick={() =>
+                    onClick={() => {
+                      const paramsToPass = serializeSearchParams({
+                        ...params,
+                        ...Object.fromEntries(searchParams.entries())
+                      })
                       navigate(
-                        formatUrl(routes.DASHBOARD, {
-                          id: config.id
-                        }),
-                        {
-                          state: { isNavigatedInsideApp: true }
-                        }
+                        `${ROUTES.V2.DASHBOARD.buildPath({ id: config.id })}?${paramsToPass}`
```

Comment:
I couldn't find the searchParams being used for anything? If I am not wrong this is used to go back to the workqueue that we navigated from but we are just going back up the history items